PropertyChanged.Fody/WarningChecker.cs

```diff
@@ -66,18 +66,23 @@ public void EmitConditionalWarning(ICustomAttributeProvider member, string messa
     {
         if (SuppressWarnings)
             return;
-        
-        if (member.HasCustomAttributes && member.CustomAttributes.ContainsAttribute("PropertyChanged.SuppressPropertyChangedWarningsAttribute"))
+
+        var suppressAttrName = "PropertyChanged.SuppressPropertyChangedWarningsAttribute";
+
+        if (member.HasCustomAttributes && member.CustomAttributes.ContainsAttribute(suppressAttrName))
             return;
-        
+
+        if (member is IMemberDefinition memberDefinition && memberDefinition.DeclaringType.GetAllCustomAttributes().ContainsAttribute(suppressAttrName))
+            return;
+
         // Get the first sequence point of the method to get an approximate location for the warning 
         var sequencePoint = member is MethodDefinition method && method.DebugInformation.HasSequencePoints
             ? method.DebugInformation.SequencePoints.FirstOrDefault()
             : null;
-        
+
         if (!message.EndsWith("."))
-            message += ".";
-        
+          message += ".";
+
         LogWarningPoint?.Invoke($"{message} You can suppress this warning with [SuppressPropertyChangedWarnings].", sequencePoint);
-    }
+  }
```

Comment:
Bad indentation, but that should get fixed on conflict resolution.

Reply:
Yeah I'm sorry.
As you can see, I integrated the `.editorconfig` file because I'm used to 2 chars indentation, so wanted to make it explicit, and it still didn't work. I had to edit all indentations manually.